open-api.yaml

```diff
@@ -2049,6 +2108,74 @@ paths:
           description: Not Found
     parameters:
       - $ref: '#/components/parameters/indexUid'
+  '/indexes/{indexUid}/settings/typo-tolerance':
+    get:
+      operationId: indexes.settings.typoTolerance.get
+      summary: Get typo tolerance configuration
+      description: |
+        Get the typo tolerance configuration of an index.
+      tags:
+        - Settings
+      security:
+        - apiKey: []
+      responses:
+        '200':
+          description: Ok
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/typoTolerance'
+        '401':
+          $ref: '#/components/responses/401'
+        '404':
+          description: Not Found
+    post:
```

Comment:
There is no PUT for the moment.

In fact, PUT requires all fields to be defined to modify the resource. While a PATCH accepts a partial update.

If we add PUT, I think that POST has no more sense because the resource already exists. We don't create anything here.

In the idea, it is either POST or PUT (All fields are required in the payload) + PATCH (Partial updates are allowed).

Today on `/settings` for example, POST allows making a partial update. 

I think the cleanest way would be to replace POST with PUT (require all fields on it) and add a PATCH route for partial udpates but that is out-of-scope for this specification. I would like to think about that for v0.28!